main.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gin-gonic/gin"
+	bpb "github.com/yarinBenisty/birthday-service/proto"
+	"google.golang.org/grpc"
+)
+
+//router that's connecting to the client
+//functions in the bd-service proto file
+type Router struct {
+	client bpb.BirthdayFunctionsClient
+}
+
+//function that creates a new client and connects
+//to the bd-service client through the proto file
+func initClientConnection() bpb.BirthdayFunctionsClient {
+	conn, err := grpc.Dial(
+		"localhost:8000",
```

Comment:
this will not always be localhost, and the port should be an environment variable

Reply:
done